patternfly/patternfly-elements
# fix: Move sassmaps into dist

Sass maps were being added to the _temp directory but that does not make them available for debugging once the code has migrated into dist.

### Related issue

- (#1243) IE11 Demo site white screening

### Preview

Link(s) to demo page(s) where this element can be viewed:
- [Link](https://deploy-preview-1244--happy-galileo-ea79c4.netlify.app/examples/) 


### What has changed and why

Add sass map files to the dist directory.


### Testing instructions

#### Test 1: Confirm files exist 
1. Compile the repository.
2. Confirm that every component includes *.css.map files.

#### Test 2: Confirm files are usable
1. Open the pfe-band demo page in IE11.
2. Confirm that sass maps are working effectively.

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
